src/test/fuzz/parse_univalue.cpp

```diff
@@ -75,6 +68,7 @@ FUZZ_TARGET_INIT(parse_univalue, initialize_parse_univalue)
     }
     try {
         (void)ParseSighashString(univalue);
+    } catch (const UniValue&) {
     } catch (const std::runtime_error&) {
```

Comment:
I think this catch now needs to go?

Reply:
I'll test this.

Reply:
Seems like `std::runtime_error` is still thrown: https://github.com/TheCharlatan/bitcoin/commits/kernelRmUnivalue_8.

Reply:
Mmh, we don't do this for any of the other `get_str()` calls, so this might be better for a follow up? Would probably also be good to re-use the text from the type error.

Reply:
> Mmh, we don't do this for any of the other get_str() calls

I think we do? E.g. https://github.com/bitcoin/bitcoin/blob/e35fb7bc48d360585b80d0c7f89ac5087c1d405e/src/rpc/util.cpp#L89-L92 and https://github.com/bitcoin/bitcoin/blob/e35fb7bc48d360585b80d0c7f89ac5087c1d405e/src/rpc/util.cpp#L1199

Reply:
Ah right, I just looked at the wrong examples :P - will push a patch for this.